src/components/molecules/AdminVenueCard/AdminVenueCard.scss

```diff
@@ -0,0 +1,79 @@
+@import "scss/constants.scss";
+
+$primary: #6f43ff;
```

Comment:
Our current primary color is different than the one in the designs, I'm going to change it to this one. But I'm not sure how this will affect our layout/theme across the website. I'll test it out and see how it goes.

Reply:
Even if we can’t change the global primary colour, you can just rename the variable when you include it there, like $primary-admin-venue-card and put it on the line after $primary.

that way it’s moved to the constants and reshareable, but won’t impact beyond the scope of this PR. Will make it a simple job to clean up in future once we know what/how we want to align all of our colours/etc.

Reply:
@denisdimitrov Looks like this still needs to be done.

Reply:
Actually, since you've updated the main `$primary` definition now, I think this just needs to be removed here.